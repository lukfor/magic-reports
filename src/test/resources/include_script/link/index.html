{{include_script("https://test.com/script.js")}}