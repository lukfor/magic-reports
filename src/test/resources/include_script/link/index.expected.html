<!-- https://test.com/script.js -->
<script src="https://test.com/script.js"></script>